specification/ai/Azure.AI.Projects/evaluations/models.tsp

```diff
@@ -78,3 +78,48 @@ model Evaluation {
   @doc("Evaluators to be used for the evaluation.")
   evaluators: Record<EvaluatorConfiguration>;
 }
+
+@doc("Definition for sampling strategy.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v_latest)
+model SamplingConfiguration {
+  @doc("Name of the sampling strategy.")
+  name: string;
+
+  @doc("Percentage of sampling per hour (0-100).")
+  samplingPercent: float;
+
+  @doc("Maximum request rate per hour (0 to 10,000).")
+  maxRequestRate: float;
+}
+
+@doc("Evaluation request for agent run.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v_latest)
+model AgentEvaluationRequest {
+  @doc("Identifier of the agent run.")
+  runId: string;
+
+  @doc("Identifier of the agent thread. This field is mandatory currently, but it will be optional in the future.")
+  threadId?: string;
+
+  @doc("Evaluators to be used for the evaluation.")
+  evaluators: Record<EvaluatorConfiguration>;
+
+  @doc("Sampling configuration for the evaluation.")
+  samplingConfiguration?: SamplingConfiguration;
+
+  @doc("Optional and temporary way to pass the AppInsights connection string to the evaluator. When this string is not null, the evaluation results will be logged to Azure AppInsights.")
+  appInsightsConnectionString?: string;
+}
+
+@doc("Long running operation response for agent evaluation.")
+@added(Versions.v2025_05_15_preview)
+@removed(Versions.v_latest)
+model AgentLongRunningOperationResponse {
```

Comment:
This is not adhering to Azure's long running operation schema. I thought this was supposed to be removed per our discussion last week?